### ISSUE TITLE
chatCellDidTapAvatar does not work

### ISSUE THREAD

**Issue by budlight** [QuickBlox/QMChatViewController-ios#48]
It seems like it should use QMImageView's imageViewDidTap but instead it uses 
- (void)handleTapGesture:(UITapGestureRecognizer *)tap {
  
  CGPoint touchPt = [tap locationInView:self];
  
  if (CGRectContainsPoint(self.avatarContainerView.frame, touchPt)) {
      [self.delegate chatCellDidTapAvatar:self];
  }

**Comment by budlight** [comment]
One way I was able to get this to work to set
[[(QMChatCell *)cell avatarView] setUserInteractionEnabled:NO];

as well as changing to:

```
if (CGRectContainsPoint(self.avatarView.frame, touchPt)) {
    [self.delegate chatCellDidTapAvatar:self];
```

**Comment by Raikerian** [comment]
Hi @budlight 

It is a known issue, thank your for reporting it.

**Comment by Raikerian** [comment]
@budlight fixed in 0.3.6.

**Comment by budlight** [comment]
ty can confirm its works
